apps/api/src/admin/participant_manager.py

```diff
@@ -17,11 +18,22 @@ class Participant(UserRecord):
     status: Status
```

Comment:
Issue: validation will fail since `Decision.Waitlisted` is a possible "status" value, so we'll need to use a `Union` here.